readyapi4j-testserver-client/src/main/java/com/smartbear/readyapi4j/testserver/RepositoryProjectExecutionRequest.java

```diff
@@ -6,6 +6,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Request object used when executing tests in server repository project
```

Comment:
or may be: 'Request object used to execute tests in/on a project available in a repository on TestServer'.